### ISSUE TITLE
[Windows] Bespoke exiting when opening sample Fubbles .bsk (filesystem encoding error)

### ISSUE THREAD

**Issue by Funto** [BespokeSynth/BespokeSynth#31]
Using the v0.0004-pre Windows installer from https://github.com/awwbees/BespokeSynth/releases/download/v0.0004-pre/Bespoke-Windows.zip

Repro steps I followed:
- On Windows 10 x64 (French, maybe related?)
- Launch Bespoke-Windows\BespokeSynth.exe
- Attach Visual Studio 2017
- Click "load state", then browse to Bespoke-Windows\data\savestate\2020-08-18_00-39_fubbles_script_demo.bsk
- BespokeSynth quits
- Excerpt from the output in Visual Studio:

'BespokeSynth.exe' (Win32): Unloaded 'C:\Windows\WinSxS\amd64_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.18362.959_none_17b2aea46d9edc89\GdiPlus.dll'
'BespokeSynth.exe' (Win32): Unloaded 'C:\Windows\System32\sendmail.dll'
The thread 0x3194 has exited with code 0 (0x0).
The thread 0x3084 has exited with code 0 (0x0).
The thread 0xffc has exited with code 0 (0x0).
'BespokeSynth.exe' (Win32): Unloaded 'C:\Windows\System32\msftedit.dll'
'BespokeSynth.exe' (Win32): Loaded 'C:\Windows\System32\zipfldr.dll'. Cannot find or open the PDB file.
The thread 0x2cfc has exited with code 0 (0x0).
**Fatal Python error: failed to get the Python codec of the filesystem encoding**
The thread 0x770 has exited with code 1 (0x1).
The thread 0x3638 has exited with code 1 (0x1).
The thread 0x3624 has exited with code 1 (0x1).

**Comment by awwbees** [comment]
ah, thank you for pointing out that you're on a french system, that's a likely culprit. I hate these sorts of bugs! I will set my system to french and see if I can reproduce it.

do you also get the issue if you spawn a "script" module(hold 's' and click "script")? if not, how about if you load a script and then click "run"?

**Comment by awwbees** [comment]
does the workaround suggested here https://github.com/pybind/pybind11/issues/2369 (setting PYTHONHOME and PYTHONPATH) help your issue?

**Comment by Funto** [comment]
So:
- yes, holding 's' + clicking "script" does trigger the issue as well
- I tried the workaround you mentionned and it worked:
 -> installed Python 3.8 64 bits since your process is 64 bits (don't know if 32 bits would have worked, but why is that still the default on python.org in 2020...)
  -> I opened cmd.exe and typed:
`
set PYTHONHOME=C:\Users\Funto\AppData\Local\Programs\Python\Python38
BespokeSynth.exe
`
then I couldn't reproduce the issue.

Note that I didn't need to set PYTHONPATH, which according to https://stackoverflow.com/questions/7850908/what-exactly-should-be-set-in-pythonpath is used for additional directories with private libraries.

Now a whole world of musical experimentation is opened to me, thanks :)
Let me know if I can help in any way.
